OffchainLabs/arbitrum-token-bridge
# refactor: remove `walletAddress` from bridge

### Summary

- Replaced `walletAddress` from `useArbTokenBridge` state.
- Replaced usage with `useAccount().address` from `wagmi`





### Steps to test

Behavior should not change compared to previous version:
- UI should react to change of accounts
- Balance should be fetched correctly, and update when user change account

### Note

Unlike `useArbTokenBridge` state, `address` from `useAccount` CAN be undefined
